BigBaseV2/src/backend/looped/player/spectate.cpp

```diff
@@ -29,9 +30,10 @@ namespace big
 
 		NETWORK::NETWORK_SET_IN_SPECTATOR_MODE(true, target);
 		HUD::SET_MINIMAP_IN_SPECTATOR_MODE(true, target);
-
+		ENTITY::SET_ENTITY_INVINCIBLE(PLAYER::PLAYER_PED_ID(), true);
```

Comment:
Remove set invincible to prevent confusion to users (they don't know their entity is made invincible / vincible).